netmap/policy.go

```diff
@@ -553,6 +553,7 @@ func writeFilterStringTo(w io.StringWriter, f netmap.Filter) error {
 func (p *PlacementPolicy) DecodeString(s string) error {
 	input := antlr.NewInputStream(s)
 	lexer := parser.NewQueryLexer(input)
+	lexer.RemoveErrorListeners()
```

Comment:
See https://github.com/nspcc-dev/neofs-sdk-go/issues/280#issuecomment-1168287995
Can we reuse existing error listener instead?

Reply:
@fyrchik pls check it out did I understand u correctly 4486898aab30039edff0d1f585a87cffb2a4ab0f

Reply:
Yes, exactly. Could you also add a test that `?REP 1` fails to be parsed?